e2e/wdio/driver/wdio.ts

```diff
@@ -290,3 +297,12 @@ export function clickAndDragElement(locationX: number, locationY: number, newLoc
         ]
     }]);
 }
+
+
+export function saveElement(selector: string, tag: string, options?: object, index: number = 0): void {
+    browser.saveElement($$(selector)[index], tag, options);
+}
+
+export function checkElement(selector: string, tag: string, options?: object, index: number = 0): any {
```

Comment:
maybe "checkElementImg" or "checkElementScreenshot" (based on the above comment)

Reply:
Done